docs/deployment.md

```diff
@@ -113,6 +114,8 @@ The following services support deploying Next.js using [`next export`](/docs/adv
 - [Cloudflare Pages](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/)
 - [Firebase](https://github.com/vercel/next.js/tree/canary/examples/with-firebase-hosting)
 - [GitHub Pages](https://github.com/vercel/next.js/tree/canary/examples/github-pages)
+- [21YunBox](https://www.21cloudbox.com/solutions/how-to-deploy-nextjs-project-in-production-server.html)
+
```

Comment:
```suggestion
```

Reply:
If it works as a server, then it probably shouldn't be listed under static only.